code/datums/crew_objective.dm

```diff
@@ -118,11 +118,12 @@
 						if(owner.current && !isdead(owner.current) && istype(get_area(owner.current),map_settings.escape_centcom)) //split this up as it was long
 							return 1
 				return 0
+
 		brigstir
 			explanation_text = "Keep Monsieur Stirstir brigged but also make sure that he comes to absolutely no harm."
 			medal_name = "Monkey Duty"
 			check_completion()
-				for(var/mob/living/carbon/human/npc/monkey/stirstir/M in mobs)
+				for(var/mob/living/carbon/human/M in mobs)
```

Comment:
Is this test-code that you accidentally left in?

Reply:
Its from a previous job objective I was persuaded to remove.

Reply:
It appears to be changing an *existing* objective

Reply:
Fixed. I think.